lib/config/secrets.ts

```diff
@@ -115,13 +122,14 @@ function replaceSecretsinObject(
 
 export function applySecretsToConfig(
   config: RenovateConfig,
-  secrets = config.secrets
+  secrets = config.secrets,
```

Comment:
Do we use this parameter at any calling function? Or do we always use this default value?

Reply:
We use it here: https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/init/merge.ts#L232